### PR TITLE
[release/1.9.0][record-minmax] Handling error when data file is not HDF5

### PR DESCRIPTION
**Pull request by llFreetimell**
This commit will add error handling code when data file is not HDF5.

ONE-DCO-1.0-Signed-off-by: Seok NamKoong <seok9311@naver.com>

### CODE REVIEW

File: compiler/record-minmax/src/HDF5Importer.h [hunk 1]
```diff
@@ -39,9 +39,12 @@ namespace record_minmax
 class HDF5Importer
 {
 public:
-  explicit HDF5Importer(const std::string &path) : _file{path, H5F_ACC_RDONLY}
+  explicit HDF5Importer(const std::string &path)
   {
-    // Do nothing
+    if (_file.isHdf5(path) == false)
+      throw std::runtime_error("Given data file is not HDF5");
```
Comment (seanshpark) [inline compiler/record-minmax/src/HDF5Importer.h:45]: (optional) "Error: Given data file is not HDF5" ?

**Review by seanshpark — APPROVED** [review]
LGTM

File: compiler/record-minmax/src/HDF5Importer.h (hunk 1 above)
Comment (llFreetimell) [reply compiler/record-minmax/src/HDF5Importer.h:45]: Master branch is already merged with same error message :(
How about applying it from 1.10.0?

**Comment by llFreetimell** [comment]
@nnfw-bot test-nncc-debug

**Comment by llFreetimell** [comment]
@nnfw-bot test nncc-release

**Comment by llFreetimell** [comment]
@nnfw-bot test nncc-debug

File: compiler/record-minmax/src/HDF5Importer.h (hunk 1 above)
Comment (seanshpark) [reply compiler/record-minmax/src/HDF5Importer.h:45]: We can apply another change to master :)

**Review by mhs4670go — APPROVED** [review]
LGTM

**Review by jinevening — APPROVED** [review]
LGTM

**Comment by llFreetimell** [comment]
@mhs4670go @seanshpark @jinevening PTAL :)

**Review by seanshpark — APPROVED** [review]
LGTM thank you!

**Review by jinevening — APPROVED** [review]

**Review by mhs4670go — APPROVED** [review]
LGTM

**Pull request merged**